src/pages/collators/Collators.tsx

```diff
@@ -16,10 +16,27 @@ enum filters {
 type Candidate = {
   owner: string;
   amount: number;
+  delegators?: number;
+};
+
+type CandidatePoolProps = {
+  owner: string;
+  delegators: number;
+};
+
+type DelegatorData = {
+  owner: string[];
+  delegators: string[];
+  id: string;
+  status: string;
+  total: string;
 };
 
 export function Collators() {
   const [candidates, setCandidates] = useState<Candidate[]>([]);
+  const [candidatePool, setCandidatePool] = useState<CandidatePoolProps[]>([]);
```

Comment:
why `CandidatePoolProps`, but not `CandidateProps`. I find it confusing. Maybe just remove `Props` from `CandidatePoolProps`?